src/store/auth/operations/loginOperation.js

```diff
@@ -25,14 +25,14 @@ export const loginOperation = createLogic({
       )
       const sessionRequestToken = sessionRequestTokenReq.data.request_token
 
-      const sessionIDReq = await axios.post(
+      const sessionIdReq = await axios.post(
```

Comment:
sessionIdReq -> response

Reply:
@VladWithCookies I can't call in just `response` because there is few requests

Reply:
it any it is not request, but response )